pkg/forwarder/transaction.go

```diff
@@ -381,5 +385,9 @@ func (t *HTTPTransaction) internalProcess(ctx context.Context, client *http.Clie
 
 // SerializeTo serializes the transaction using TransactionsSerializer
 func (t *HTTPTransaction) SerializeTo(serializer *TransactionsSerializer) error {
-	return serializer.Add(t)
+	if t.storableOnDisk {
+		return serializer.Add(t)
+	}
+	log.Debug("The transaction is not stored on disk because `storableOnDisk` is false.")
```

Comment:
Is this too verbose? Maybe more a trace-level statement? (I'm not sure just thinking out loud). Also, this doesn't tell us much about the transaction that was not stored to disk... Anyway, please let me known your thoughts.

Reply:
I have changed from `Debug` to `Trace` (it can be useful for the QA of this development).